README.md

```diff
@@ -1,2 +1,145 @@
-# Hello_Microverse
-Set up the first project on Github
+<a name="Hello_Microverse!"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Setting up Hello_Microverse! locally</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello_Microverse!] <a name="about-project"></a>
+
+**[Hello_Microverse!]** is a a project that allows you understand how to setting up your project locally and upload this to github as a repository, fixing the possible issues that you can get in the process.
+
+## 🛠 Built With <a name="Visual Studio Code"></a>
+
+### Tech Stack <a name="Viual Studio Code"></a>
+
+### Key Features <a name="key-features"></a>
+
+- **[Github]**
+- **[Visual_Studio_Code]**
+- **[Linters]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://github.com/adelphinsucasa/Hello_Microverse.git)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] [OPTIONAL] If you have not deployment link of your project it would be better remove the this part 😃

Reply:
Great, you are right. I have removed this section. Thank you